src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -1950,6 +1950,24 @@ Well{0} entered with 'FIELD' parent group:
         }
     }
 
+    void Schedule::handleWINJMULT(Opm::Schedule::HandlerContext& handlerContext) {
```

Comment:
Tiny nit, but we attempt to keep the `handle*()` functions in alphabetically sorted order in this file.  Would you mind placing the function between [`WHISTCTL`](https://github.com/OPM/opm-common/blob/dd3492606237ad37a3c99a70ab13445d1abc662f/src/opm/input/eclipse/Schedule/KeywordHandlers.cpp#L1755) and [`WINJTEMP`](https://github.com/OPM/opm-common/blob/dd3492606237ad37a3c99a70ab13445d1abc662f/src/opm/input/eclipse/Schedule/KeywordHandlers.cpp#L1789) please?  Similarly for the entry in the [`handler_functions`](https://github.com/OPM/opm-common/blob/dd3492606237ad37a3c99a70ab13445d1abc662f/src/opm/input/eclipse/Schedule/KeywordHandlers.cpp#L2394) table.